TextToSpeech/impl/TTSConfiguration.h

```diff
@@ -31,6 +31,8 @@ class TTSConfiguration {
     bool setRate(const uint8_t rate);
     bool setPrimVolDuck(const int8_t primvolduck);
     bool setSATPluginCallsign(const std::string callsign);
+    bool setAccessList(const string &key,const string &value);
```

Comment:
Why is ACL part of configuration and not TTSManager?

Reply:
addressed on https://github.com/rdkcentral/rdkservices/pull/4359